indexer/documents/conv.go

```diff
@@ -389,6 +389,7 @@ func ConvChainInfo(chainInfo *types.ChainInfo) *EsChainInfo {
 		Mainnet:    chainInfo.Id.Mainnet,
 		Consensus:  chainInfo.Id.Consensus,
 		Version:    uint64(chainInfo.Id.Version),
+		Hardfork: 	chainInfo.Hardfork,
```

Comment:
Why to save this to the db if the value changes and depends on the block number?

When processing an old transaction, the hardfork of that transaction can be different from the current hardfork of the chain